default/python/AI/AIstate.py

```diff
@@ -114,6 +123,7 @@ def __init__(self, aggression):
         # unique ids for turns.  {turn: uid}
         self.turn_uids = {}
 
+        # see AIstate docstring re importance of int cast for aggression
```

Comment:
` re ` ?

Reply:
@Cjkjvfnby 

> about; concerning.
"I saw the deputy re the incident"

the above example is [from here](https://www.google.com/search?q=dictionary%3A+re&oq=dictionary%3A+re&aqs=chrome..69i57j69i58.3087j0j7&sourceid=chrome&ie=UTF-8)

you can think of it as akin to an abbreviation for "regarding", although I don't think that's what it actually is.